pkg/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -780,3 +785,23 @@ func checkReplicationPrivileges(
 	}
 	return nil
 }
+
+// ldrDestTableDescTransformerFactory returns a transformer function that is
+// applied to created destination table descriptors before they are written to
+// the descriptors table.
+func ldrDestTableDescTransformerFactory(
+	unidirectional bool,
+) func(descpb.TableDescriptor, *tabledesc.Mutable) {
+	if !unidirectional {
+		return nil
+	}
+
+	return func(_ descpb.TableDescriptor, td *tabledesc.Mutable) {
+		for idx := range td.Columns {
+			// Unidrectional LDR streams support sequences by simply ignoring the
+			// sequence dependency on the destination table and replicating the raw
+			// values in the columns.
+			td.Columns[idx].UsesSequenceIds = []descpb.ID{}
```

Comment:
While this does work, I think some additional clean up may need to be done. When running `SHOW CREATE TABLE` on the replicated destination table, the `CREATE` command still shows `nextval`:

```
  table_name |                      create_statement
-------------+--------------------------------------------------------------
  foocpy     | CREATE TABLE public.foocpy (
             |     x INT8 NULL DEFAULT nextval('public.foocpy'::REGCLASS),
             |     rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
             |     CONSTRAINT foo_pkey PRIMARY KEY (rowid ASC)
             | ) WITH (schema_locked = true);
```